mappings/net/minecraft/block/dispenser/ProjectileDispenserBehavior.mapping

```diff
@@ -1,7 +1,16 @@
 CLASS net/minecraft/class_2965 net/minecraft/block/dispenser/ProjectileDispenserBehavior
+	COMMENT A dispenser behavior that spawns a projectile with velocity in front of the dispenser.
 	METHOD method_12844 createProjectile (Lnet/minecraft/class_1937;Lnet/minecraft/class_2374;Lnet/minecraft/class_1799;)Lnet/minecraft/class_1676;
+		COMMENT Creates the entity that will be spawned in front of the dispenser.
+		COMMENT
+		COMMENT @return the created projectile
 		ARG 1 world
+			COMMENT the world the projectile will spawn in
 		ARG 2 position
+			COMMENT the output location of the dispenser
 		ARG 3 stack
+			COMMENT the stack that the dispenser will consume
 	METHOD method_12845 getVariation ()F
+		COMMENT {@return the variation of a projectile's velocity when spawned}
 	METHOD method_12846 getForce ()F
+		COMMENT {@return the force of a projectile's velocity when spawned}
```

Comment:
Hmm, might consider describe how these variation and force work, like if they are a linear or a quadratic factor to the default velocity

Reply:
This could be documented in `setVelocity` instead.